packages/common/src/schemas/documents/default-context-schema.json

```diff
@@ -167,7 +167,15 @@
                   "additionalProperties": false
                 }
               },
-              "required": ["provider", "uri", "type"],
+              "required": ["provider", "type"],
+              "oneOf": [
+                {
+                  "required": ["ballerineFileId"]
+                },
+                {
+                  "required": ["uri"]
+                }
+              ],
               "additionalProperties": false
```

Comment:
i've tested the flow its seems that we just need to create a migration or update the db directly for relevant contextSchema which isnt was align even i reset the db (so maybe seed changes is also required)